src/OrbitGl/TimeGraph.cpp

```diff
@@ -562,7 +563,8 @@ void TimeGraph::UpdateChildrenPosAndContainerSize() {
   float current_pos_y = GetPos()[1];
   for (orbit_gl::CaptureViewElement* child : GetNonHiddenChildren()) {
     child->SetPos(GetPos()[0], current_pos_y);
-    current_pos_y += child->GetHeight();
+    current_pos_y += child->GetHeight() + layout_.GetSpaceBetweenContainers();
```

Comment:
It's expected behavior that this space is added in between all children of `TimeGraph`, not specifically only between the timeline UI and the track container?